app/src/main/java/com/x8bit/bitwarden/data/platform/repository/EnvironmentRepositoryImpl.kt

```diff
@@ -30,6 +32,10 @@ class EnvironmentRepositoryImpl(
             .toEnvironmentUrlsOrDefault()
         set(value) {
             environmentDiskSource.preAuthEnvironmentUrlData = value.environmentUrlData
+            scope.launch {
```

Comment:
Firing off a network request is a bit of a heavy side effect to have directly in the setter. A better option would be to subscribe to the `environmentStateFlow`, this could be done in the `init` block of the `ServerConfigRepositoryImpl`.